Sources/KavaPlugin.swift

```diff
@@ -284,8 +284,13 @@ let playbackPoints: [KavaPlugin.KavaEventType] = [KavaPlugin.KavaEventType.playR
             guard let data = response.data, let responseJson = JSON(data).dictionary else { return }
             
             if (self.config.sessionStartTime == nil) {
-                guard let sessionStartTime = responseJson["time"]?.number else { return }
-                self.config.sessionStartTime = sessionStartTime.stringValue
+                if let sessionStartTime = responseJson["time"]?.number {
+                    // Response returned in JSON format
+                    self.config.sessionStartTime = sessionStartTime.stringValue
+                } else {
+                    // Response returned in String format
+                    self.config.sessionStartTime = responseJson["time"]?.string
```

Comment:
In string(old format) response there shouldn’t be any data fields in the response, so you shouldn’t be expecting a field called _time_